test/integration/model/bulk-create.test.js

```diff
@@ -310,7 +310,6 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         const expectedValidationError = 'Validation len on code failed';
         const expectedNotNullError = 'notNull Violation: Task.name cannot be null';
 
-        expect(error).to.be.instanceof(AggregateError);
```

Comment:
`instanceof` does not work with transpiled code. See https://www.chaijs.com/api/bdd/#method_instanceof and https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes#extending-built-ins-like-error-array-and-map-may-no-longer-work for more details